src/controllers/v1/match2/index.ts

```diff
@@ -317,8 +317,11 @@ export class Match2Controller extends Controller {
         if (!ownsDemandA && !ownsDemandB) throw new BadRequest(`You do not own an acceptable demand`)
         return acceptAB()
       case 'acceptedA':
+        if (!ownsDemandB) throw new BadRequest(`You do not own an acceptable demand`)
+        if (match2.replaces) return acceptRematch()
+        return acceptFinal()
       case 'acceptedB':
-        if (!ownsDemandB || !ownsDemandA) throw new BadRequest(`You do not own an acceptable demand`)
```

Comment:
this logic meant no member could ever finally accept a match2 unless they owned both demands... this PR changes it back to what it used to be